code/ZAS/Fire.dm

```diff
@@ -52,21 +53,21 @@ turf/proc/hotspot_expose(exposed_temperature, exposed_volume, soh = 0)
 				T.fire.firelevel = firelevel
 			else
 				var/obj/effect/decal/cleanable/liquid_fuel/fuel = locate() in T
-				fire_tiles -= T
-				fuel_objs -= fuel
+				LAZYREMOVE(fire_tiles, T)
+				LAZYREMOVE(fuel_objs, fuel)
 	else
 		for(var/turf/simulated/T in fire_tiles)
 			if(istype(T.fire))
 				T.fire.RemoveFire()
 			T.fire = null
-		fire_tiles.Cut()
-		fuel_objs.Cut()
+		LAZYCLEARLIST(fire_tiles)
```

Comment:
I was just leaving the lists hanging around as it's fairly likely they'll be used again after they've been created.